sys/net/gnrc/link_layer/netdev2/gnrc_netdev2.c

```diff
@@ -81,6 +81,14 @@ static void _event_cb(netdev2_t *dev, netdev2_event_t event)
                 dev->stats.tx_success++;
                 break;
 #endif
+#ifdef MODULE_GNRC_PPP
+            case NETDEV2_EVENT_LINK_UP:
```

Comment:
The fact alone that you only call these functions here should give you a hint that you really do not need to extend `netdev` by these functions ;-).